docs/index.rst

```diff
@@ -185,52 +119,7 @@ are using a different package index.
 
 For completeness, its usage:
 
-.. code-block:: text
-
-    usage: twine register [-h] -r REPOSITORY [--repository-url REPOSITORY_URL]
-                          [-u USERNAME] [-p PASSWORD] [-c COMMENT]
-                          [--config-file CONFIG_FILE] [--cert path]
-                          [--client-cert path]
-                          package
-
-    positional arguments:
-      package               File from which we read the package metadata.
-
-    optional arguments:
-      -h, --help            show this help message and exit
-      -r REPOSITORY, --repository REPOSITORY
-                            The repository (package index) to register the package
-                            to. Should be a section in the config file. (Can also
-                            be set via TWINE_REPOSITORY environment variable.)
-                            Initial package registration no longer necessary on
-                            pypi.org:
-                            https://packaging.python.org/guides/migrating-to-pypi-
-                            org/
-      --repository-url REPOSITORY_URL
-                            The repository (package index) URL to register the
-                            package to. This overrides --repository. (Can also be
-                            set via TWINE_REPOSITORY_URL environment variable.)
-      -u USERNAME, --username USERNAME
-                            The username to authenticate to the repository
-                            (package index) as. (Can also be set via
-                            TWINE_USERNAME environment variable.)
-      -p PASSWORD, --password PASSWORD
-                            The password to authenticate to the repository
-                            (package index) with. (Can also be set via
-                            TWINE_PASSWORD environment variable.)
-      --non-interactive     Do not interactively prompt for username/password
-                            if the required credentials are missing. (Can also
-                            be set via TWINE_NON_INTERACTIVE environment
-                            variable.)
-      -c COMMENT, --comment COMMENT
-                            The comment to include with the distribution file.
-      --config-file CONFIG_FILE
-                            The .pypirc config file to use.
-      --cert path           Path to alternate CA bundle (can also be set via
-                            TWINE_CERT environment variable).
-      --client-cert path    Path to SSL client certificate, a single file
-                            containing the private key and the certificate in PEM
-                            format.
+.. program-output:: twine register -h
```

Comment:
Could we instead use https://pypi.org/project/sphinx-argparse-cli/ to get a easier to read and linkable output?

Reply:
We may need some opinions here. @bhrutledge @sigmavirus24

Reply:
I'd like to see a working example. Also, the original issue #793 also had some other suggestions; I don't know if one is more appropriate for Twine. I'd prefer something that doesn't require code changes.

Reply:
Created https://twine--800.org.readthedocs.build/en/800/#commands as POC. Due to the design of twine it does require code changes, no way around it 🤷 .